langkit/support/langkit_support-symbols.ads

```diff
@@ -83,6 +102,8 @@ package Langkit_Support.Symbols is
 
 private
 
+   type Thin_Symbol is new Natural;
```

Comment:
Given that the very goal of this type is to be 32-bits wide, I think we should use Interfaces.Unsigned_32 instead of Natural